test/unit/run-summary.test.js

```diff
@@ -33,7 +33,8 @@ describe('run summary', function () {
     it('should have the relevant top-level data structures', function () {
         var summary = new Summary(new EventEmitter());
 
-        expect(_.keys(summary).sort()).to.eql(['collection', 'environment', 'globals', 'run'].sort());
+        expect(_.keys(summary).sort()).to.eql(['collection', 'environment', 'globals', 'reporters',
+            'run', 'skipExecutionResponse'].sort());
```

Comment:
I don't think we should be changing the API here. This test should not need to be changed.